Cannot find module 'graphql'

**Version**: 0.14.0-alpha6

The CLI is not working in alpha6 because `graphql` is missing.

```
❯ npx graphback-cli@0.14.0-alpha6 init fullstack-issue           
npx: installed 521 in 28.088s
Cannot find module 'graphql'
Require stack:
- /home/ephelan/.npm/_npx/396521/lib/node_modules/graphback-cli/node_modules/@graphback/core/dist/crud/mappingHelpers.js
- /home/ephelan/.npm/_npx/396521/lib/node_modules/graphback-cli/node_modules/@graphback/core/dist/crud/index.js
- /home/ephelan/.npm/_npx/396521/lib/node_modules/graphback-cli/node_modules/@graphback/core/dist/index.js
- /home/ephelan/.npm/_npx/396521/lib/node_modules/graphback-cli/node_modules/graphback/dist/index.js
- /home/ephelan/.npm/_npx/396521/lib/node_modules/graphback-cli/dist/components/generate.js
- /home/ephelan/.npm/_npx/396521/lib/node_modules/graphback-cli/dist/components/index.js
- /home/ephelan/.npm/_npx/396521/lib/node_modules/graphback-cli/dist/index.js
```

This is because `generate` command uses the graphback package, which in turn looks for a peer dependency.

--- a/packages/graphback-cli/src/config/graphbackExtension.ts
+++ b/packages/graphback-cli/src/config/graphbackExtension.ts
@@ -1,5 +1,5 @@
 import { CodeFileLoader } from '@graphql-toolkit/code-file-loader';
-import { GraphQLExtensionDeclaration, loadConfig } from 'graphql-config';
+import { GraphQLExtensionDeclaration } from 'graphql-config';
 
 
 export const graphbackExtension = 'graphback';

--- a/packages/graphback-cli/src/index.ts
+++ b/packages/graphback-cli/src/index.ts
@@ -1,7 +1,6 @@
 #!/usr/bin/env node
 
 import * as yargs from 'yargs';
-export * from './components'
 export * from './config/graphbackExtension'
 
 if (require.main === module) {

--- a/packages/graphback-cli/src/index.ts
+++ b/packages/graphback-cli/src/index.ts
@@ -1,7 +1,6 @@
 #!/usr/bin/env node
 
 import * as yargs from 'yargs';
-export * from './config/graphbackExtension'
 
 if (require.main === module) {
   // eslint-disable-next-line no-unused-expressions

